src/Components/TicTacToe/Board.js

```diff
@@ -0,0 +1,27 @@
+import React from "react"
+import Square from "./Square"
+
+
+export default function Board({squares, size, onClick}) {
+
+  const extendedStyle = {
+    ...style,
+    gridTemplateRows: `repeat(${size}, 1fr)`,
+    gridTemplateColumns: `repeat(${size}, 1fr)`
+  }
+
+  return (
+    <div style={extendedStyle}>
+      {squares.map((square, i) => (
+        <Square value={square} key={i} onClick={() => onClick(i)}/>
+      )
+      )}
+    </div>
+  );
+}
+
+const style = {
+  display: "grid",
+  width: "500px",
+  height: "500px",
+};
```

Comment:
I think it's better to keep styles in separate css files, at least where possible. Take a look at CSS modules (I've used them in other components); they offer quite nice integration between js and css.